elki-core-util/src/main/java/de/lmu/ifi/dbs/elki/utilities/optionhandling/parameterization/ChainedParameterization.java

```diff
@@ -58,9 +59,7 @@
    * @param ps Parameterizations
    */
   public ChainedParameterization(Parameterization... ps) {
-    for(Parameterization p : ps) {
-      chain.add(p);
-    }
+    Collections.addAll(chain, ps);
```

Comment:
I would personally prefer the Collections way because it shows the intention more clearly in my opinion, but the `for`-loop is already pretty clear in this case as well, so I guess the difference is small.